### ISSUE TITLE
Cri-containerd can not work normally with the default-config

### ISSUE THREAD

**Issue by miaoyq** [containerd/cri-containerd#633]
Set cri-containerd config file:
```
$ cri-containerd default-config > /etc/cri-containerd/config.toml
```
Restart cri-containerd service:
```
$ systemctl restart cri-containerd
```
Check the status of cri-containerd:
```
$ systemctl status cri-containerd.service
● cri-containerd.service - Kubernetes containerd CRI shim
   Loaded: loaded (/etc/systemd/system/cri-containerd.service; disabled; vendor preset: disabled)
   Active: activating (auto-restart) (Result: exit-code) since Fri 2018-03-02 17:32:08 CST; 4s ago
  Process: 13958 ExecStart=/usr/local/bin/cri-containerd (code=exited, status=1/FAILURE)
 Main PID: 13958 (code=exited, status=1/FAILURE)

Mar 02 17:32:08 node-172-20-128-76 cri-containerd[13958]: --root-dir string                    Root directory path for cri-containerd manag...nerd")
Mar 02 17:32:08 node-172-20-128-76 cri-containerd[13958]: --sandbox-image string               The image used by sandbox container. (defaul...:3.0")
Mar 02 17:32:08 node-172-20-128-76 cri-containerd[13958]: --socket-path string                 Path to the socket which cri-containerd serv...sock")
Mar 02 17:32:08 node-172-20-128-76 cri-containerd[13958]: --stats-collect-period int           The period (in seconds) of snapshots stats c...lt 10)
Mar 02 17:32:08 node-172-20-128-76 cri-containerd[13958]: --stream-addr string                 The ip address streaming server is listening...ified.
Mar 02 17:32:08 node-172-20-128-76 cri-containerd[13958]: --stream-port string                 The port streaming server is listening on. (...0010")
Mar 02 17:32:08 node-172-20-128-76 cri-containerd[13958]: --systemd-cgroup                     Enables systemd cgroup support. By default n...abled.
Mar 02 17:32:08 node-172-20-128-76 cri-containerd[13958]: Use "cri-containerd [command] --help" for more information about a command.
Mar 02 17:32:08 node-172-20-128-76 systemd[1]: Unit cri-containerd.service entered failed state.
Mar 02 17:32:08 node-172-20-128-76 systemd[1]: cri-containerd.service failed.
Hint: Some lines were ellipsized, use -l to show in full.
```

 #557 caused the problem.
@Random-Liu

**Comment by mikebrow** [comment]
Error: failed to run cri-containerd with grpc server: failed to initialize containerd client with endpoint "": failed to dial "": dial unix: missing address

I'll push a fix..

**Comment by Random-Liu** [comment]
I use `sudo cri-containerd` all the time.

**Comment by Random-Liu** [comment]
Fixed by #634

**Comment by Random-Liu** [comment]
@miaoyq Feel free to reopen if you see this again.
